packages/server/beaver_triple_service/cmd/generate_jwt.go

```diff
@@ -0,0 +1,175 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	jwt_types "github.com/acompany-develop/QuickMPC/packages/server/beaver_triple_service/jwt"
+)
+
+type Options struct {
+    file_path  string
+    output_dir string
```

Comment:
Could you use tab for indent?

Reply:
Corrected with [bf2aa41](https://github.com/acompany-develop/QuickMPC/pull/123/commits/bf2aa416419195f2fc52f4f7c768dd4c4523ebeb)